testutil/types.go

```diff
@@ -24,3 +51,23 @@ const (
 	UpgradeModuleName      = "upgrade"
 	EpochsModuleName       = "epochs"
 )
+
+// NetworkI is an interface for a network of validators.
+// It is used to abstract over the different network types (in-process, docker, etc.).
+// if used there is a requirement to expose query and tx client for the nodes
+type NetworkI interface {
```

Comment:
how come this was moved here? circular dependency

Reply:
I moved the `network` package from `testutil` to `tests`, but since these interfaces are used in `testutil/cli`, I moved them back to `testutil`. To avoid circular dependency, I will relocate them to a new `network` package within `testutil` if necessary.

Reply:
if we can leave them where they were that would be good. less api breakage.